sn_registers/src/register.rs

```diff
@@ -196,11 +188,6 @@ impl Register {
         self.crdt.get(hash).ok_or(Error::NoSuchEntry(hash))
     }
 
-    /// Return a value corresponding to the provided 'hash', if present.
-    pub fn get_cloned(&self, hash: EntryHash) -> Result<Entry> {
-        self.crdt.get(hash).cloned().ok_or(Error::NoSuchEntry(hash))
-    }
-
     /// Read the last entry, or entries when there are branches, if the register is not empty.
     pub fn read(&self) -> BTreeSet<(EntryHash, Entry)> {
```

Comment:
we could replace the function name with an overview of what the comment above is saying, removing the need for the comment and making the function name more descriptive